package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=6.4.0"
   },
   "puppeteer": {
-    "chromium_revision": "681777"
+    "chromium_revision": "686378"
```

Comment:
we generally do rolls as their own commits, as they can lots of unforeseen side effects.